db/migrate/20210902094654_change_contributor_phone_number_column_name.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class ChangeContributorPhoneNumberColumnName < ActiveRecord::Migration[6.1]
+  def change
+    rename_column :contributors, :phone_number, :signal_phone_number
```

Comment:
Hm I don't get it. What comes first? Do we migrate all `phone_number`s we had to `signal_phone_number`s? I thought one column is for internal use only (curated by users/editors) and one is the number that the user has entered.

Reply:
The phone column is for internal use only and to make that clearer @phoeinx renamed the phone_number column (which is used for signal) to signal_phone_number.